compliance/evidence.py

```diff
@@ -81,22 +79,38 @@ def from_locker(cls, path, locker):
         path = substitute_config(path)
         return cls(locker.get_evidence(path))
 
+    @property
+    def rootdir(self):
```

Comment:
How’s this set?

Reply:
It's something that's always been set as a property on each of the base evidence classes but we never officially set it in the _BaseEvidence class which seemed wrong to me.  This just handles that bit.  See: https://github.com/ComplianceAsCode/auditree-framework/blob/add-evidence-casting-to-decorators-and-ctx-mgr/compliance/evidence.py#L142 as an example.